MatrixKit/Models/Contact/MXKContactManager.m

```diff
@@ -1522,10 +1526,18 @@ - (void)observeValueForKeyPath:(NSString *)keyPath ofObject:(id)object change:(N
 
 #pragma mark - file caches
 
-static NSString *matrixContactsFile = @"matrixContacts";
-static NSString *matrixIDsDictFile = @"matrixIDsDict";
-static NSString *localContactsFile = @"localContacts";
-static NSString *contactsBookInfoFile = @"contacts";
+static NSString *MXKContactManagerDomain = @"org.matrix.MatrixKit.MXKContactManager";
+static NSInteger MXContactManagerEncryptionDelegateNotReady = -1;
+
+static NSString *matrixContactsFileOld = @"matrixContacts";
+static NSString *matrixIDsDictFileOld = @"matrixIDsDict";
+static NSString *localContactsFileOld = @"localContacts";
+static NSString *contactsBookInfoFileOld = @"contacts";
+
+static NSString *matrixContactsFile = @"matrixContactsCiph";
```

Comment:
The `Ciph` suffix may be misleading as the encryption is optional.
Why not `matrixContactsV2`?